### PR TITLE
Use JACK define constant

### PR DESCRIPTION
**Pull request by ann0see**
Fixes: https://github.com/jamulussoftware/jamulus/issues/1811

After looking a bit at the code, I do not think that it actually has a lot of benefit. Please start %1 to restart %2 ??? This doesn't really help translators in my opinion... There are some other fixes, however, they can be merged in a different PR...

### CODE REVIEW

File: linux/sound.cpp [hunk 1]
```diff
@@ -61,9 +62,10 @@ void CSound::OpenJack ( const bool bNoAutoJackConnect, const char* jackClientNam
     // check sample rate, if not correct, just fire error
     if ( jack_get_sample_rate ( pJackClient ) != SYSTEM_SAMPLE_RATE_HZ )
     {
-        throw CGenErr ( QString ( tr ( "JACK isn't running at a sample rate of <b>%1 Hz</b>. Please use "
+        throw CGenErr ( QString ( tr ( "%! isn't running at a sample rate of <b>%2 Hz</b>. Please use "
```
Comment (ann0see) [inline linux/sound.cpp:65]: Will not work. Is a typo...

File: linux/sound.cpp [hunk 2]
```diff
@@ -207,11 +210,11 @@ int CSound::Init ( const int /* iNewPrefMonoBufferSize */ )
     // without a Jack server, Jamulus makes no sense to run, throw an error message
     if ( bJackWasShutDown )
     {
-        throw CGenErr ( QString ( tr ( "JACK was shut down. %1 "
-                                       "requires JACK to run. Please restart %1 to "
-                                       "start JACK again. "
-                                       "the JACK output." ) )
```
Comment (ann0see) [inline linux/sound.cpp:213]: To do in another PR.

File: linux/sound.cpp [hunk 3]
```diff
@@ -207,11 +210,11 @@ int CSound::Init ( const int /* iNewPrefMonoBufferSize */ )
...
-                            .arg ( APP_NAME ) );
+        throw CGenErr ( QString ( tr ( "%1 was shut down. %2 "
+                                       "requires %1 to run. Please restart %2 to "
+                                       "start %1 again." ) )
```
Comment (ann0see) [inline linux/sound.cpp:215]: I think this *removes* context.

**Review by ann0see — COMMENTED** [review]
Not happy... therefore I stopped.

**Comment by pljones** [comment]
Agreed.  Now I see the result, it's not useful.

**Comment by ann0see** [comment]
Ok. So closing this.
